### PR TITLE
CVE-2019-19334 init

### PR DESCRIPTION
**Pull request by rehakm**

### CODE REVIEW

File: 2019/19xxx/CVE-2019-19334.json [hunk 1]
```diff
@@ -4,15 +4,68 @@
...
+                    {
+                        "lang": "eng",
+                        "value": "CWE-121"
+                    }
+                ]
+            }
+        ]
+    },
+    "references": {
+        "reference_data": [
```
Comment (iamleot) [inline 2019/19xxx/CVE-2019-19334.json]: Please add upstream commit to references:

https://github.com/CESNET/libyang/commit/6980afae2ff9fcd6d67508b0a3f694d75fd059d6

File: 2019/19xxx/CVE-2019-19334.json [hunk 2]
```diff
@@ -4,15 +4,68 @@
...
+                "refsource": "CONFIRM"
+            }
+        ]
     },
     "description": {
         "description_data": [
             {
                 "lang": "eng",
-                "value": "** RESERVED ** This candidate has been reserved by an organization or individual that will use it when announcing a new security problem. When the candidate has been publicized, the details for this candidate will be provided."
+                "value": "A stack-based buffer overflow was discovered in, libyang all versions before 1.0-r5, in the way libyang parses YANG files with a leaf of type \"identityref\". An application that uses libyang to parse untrusted YANG files may be vulnerable to this flaw, which would allow an attacker to cause a denial of service or possibly gain code execution."
```
Comment (iamleot) [inline 2019/19xxx/CVE-2019-19334.json]: Minor grammar nitpick: please change from:

````
A stack-based buffer overflow was discovered in, libyang all versions before 1.0-r5, in the way libyang parses YANG files with a leaf of type \"identityref\".
````

to:

````
In all versions of libyang before 1.0-r5, a stack-based buffer overflow was discovered in the way libyang parses YANG files with a leaf of type \"identityref\".
````

Comment (rehakm) [reply 2019/19xxx/CVE-2019-19334.json]: Hi there, description changed.

File: 2019/19xxx/CVE-2019-19334.json (hunk 1 above)
Comment (rehakm) [reply 2019/19xxx/CVE-2019-19334.json]: Reference added.

**Pull request merged**